### DISCUSSION TITLE
Unable to opt out of SWC

### DISCUSSION THREAD

**Discussion by mathiasritter** [vercel/next.js#44959] [Help]
Hi, I am using nextjs 13.1.2 and want to opt out of using SWC. I followed [the instructions](https://nextjs.org/docs/messages/failed-loading-swc#possible-ways-to-fix-it) and added both a `.babelrc` and a `next.config.mjs` to my project:

`.babelrc`
```json
{
  "presets": ["next/babel"]
}
```

`next.config.mjs`
```mjs
/**
 * @type {import('next').NextConfig}
 */
const nextConfig = {
    swcMinify: false,
};

export default nextConfig;
```

However, when building, it still tries to download and load the SWC binary:
```
info  - Downloading WASM swc package...
warn  - Attempted to load @next/swc-linux-x64-gnu, but an error occurred: /lib64/libc.so.6: version `GLIBC_2.15' not found (required by node_modules/@next/swc-linux-x64-gnu/next-swc.linux-x64-gnu.node)
warn  - Attempted to load @next/swc-linux-x64-gnux32, but it was not installed
warn  - Attempted to load @next/swc-linux-x64-musl, but an error occurred: libc.musl-x86_64.so.1: cannot open shared object file: No such file or directory
error - Failed to load SWC binary for linux/x64, see more info here: https://nextjs.org/docs/messages/failed-loading-swc
```

Why does opting out of SWC not work?

**Comment by icyJoseph** [comment]
Did you remove `.next` and `node_modules` and reinstall everything again? (npm ci, or just yarn).

**Reply by mathiasritter** [reply]
Yes, I tried that but same result. Also deleted `package-lock.json`.

**Comment by avshyz** [comment]
I'm encountering the same problem.
```
 Failed to download swc package from https://registry.npmjs.org/@next/swc-linux-arm-gnueabihf/-/swc-linux-arm-gnueabihf-14.1.0.tgz
```
This is due to the fact `@next/swc-linux-arm-gnueabihf` isn't updated as frequently as `nextjs`.
I also tried to follow the guide to opt out of swc, but it didn't work.
